google/cloud/logging_v2/handlers/structured_log.py

```diff
@@ -63,10 +63,18 @@ class StructuredLogHandler(logging.StreamHandler):
     """
 
     def __init__(
-        self, *, labels=None, stream=None, project_id=None, json_encoder_cls=None
+        self,
+        *,
+        labels=None,
+        stream=None,
+        project_id=None,
+        json_encoder_cls=None,
+        **kwargs
     ):
         """
         Args:
+            name (Optional[str]): Placeholder for setup_logging consistency. Does nothing.
+            resource (Optional[dict]): Placeholder for setup_logging consistency. Does nothing.
```

Comment:
nit: you can probably remove these comments about the args. 

Maybe add a line for `**kwargs` to the Args list here if you want to mention that others are ignored, but that's optional